hello-app-iap/README.md

```diff
@@ -0,0 +1,20 @@
+# ESP Sample App
+
+Displays the email address of the logged in user from the value of the `x-goog-authenticated-user-email` header inserted by the Identity-Aware Proxy.
+
+## Building
```

Comment:
Delete this and just say "this image is available at `gcr.io/google-samples/hello-app-iap:lastest`